FosServer/src/main/java/com/example/fostracker/models/Verification.java

```diff
@@ -0,0 +1,66 @@
+package com.example.fostracker.models;
+
+import java.sql.Timestamp;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+/** Serves as a model for store objects that will be used in to communicate information about GPay Merchants' stores
+ * and businesses */
+public class Verification {
+    String agentID;
+    String storeID;
+    Coordinates verificationCoordinates;
+    String verificationStatus;
+    Timestamp verificationDateTime;
+
+    /**
+     * @param agentID
+     * @param storeID
+     * @param verificationCoordinates
+     * @param verificationStatus
```

Comment:
Please add details about these params.
'@ param' should be used to add more documentation about the variables

Reply:
Hey, I can see that you made these changes in the commit above, but able to see it in the PR. Did you push these commits into this branch?

Reply:
Yes, they have been pushed